app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.test.tsx

```diff
@@ -42,8 +42,8 @@ describe('ConfigurationSteps Page Suite', () => {
 		const ConnectServerStep = screen.getByText('Connect server');
 
 		// Visited steps are styled via parent container
-		expect(InstallJenkinsStepParentContainer).toHaveAttribute('style', 'color: rgb(23, 43, 77); font-weight: 600;');
-		expect(CreateServerStepParentContainer).toHaveAttribute('style', 'color: rgb(23, 43, 77); font-weight: 600;');
+		expect(InstallJenkinsStepParentContainer).toHaveAttribute('style', 'color: rgb(165, 173, 186); font-weight: 600;');
+		expect(CreateServerStepParentContainer).toHaveAttribute('style', 'color: rgb(165, 173, 186); font-weight: 600;');
```

Comment:
/nitpick It feels like there should be a better way to test stylings ... or not test them at all. At least collect the available styles in properly named constants so we don't have to type the rgb colors again all the time.

Reply:
@mboudreau and I discussed this during another PR (https://github.com/atlassian/jenkins-for-jira/pull/22#discussion_r929534030) via slack. Essentially, this is the only viable way of testing the component we can see. I think it's a good idea to add these into vars for better readability.